README.md

```diff
@@ -37,6 +37,7 @@ installation and postprocessing procedures. Its features include:
   * Kinetic energy-preserving and entropy-stable methods based on flux differencing
   * Entropy-stable shock capturing
   * Positivity-preserving limiting
+  * Subcell invariant domain-preserving (IDP) limiting
```

Comment:
Please update https://github.com/trixi-framework/Trixi.jl/blob/main/docs/src/index.md accordingly

Reply:
Done in [eb84655](https://github.com/trixi-framework/Trixi.jl/pull/2084/commits/eb84655eb7a84b4f8f0812220059da0c79cbc2ae)